tasks/apt.yml

```diff
@@ -3,8 +3,20 @@
   changed_when: false
 
 - name: Install system packages
-  apt: name={{ item }} state=latest
-  notify: 
+  apt:
+    name:
+        - sudo
+        - software-properties-common
```

Comment:
looks like you're using 4 spaces indentation here (and in other sections below as well), while the rest of the playbook uses 2 spaces, could you make it consistent please?